qa/tasks/mgr/dashboard/test_auth.py

```diff
@@ -335,7 +344,8 @@ def test_check_wo_token(self):
         self.assertStatus(200)
         data = self.jsonBody()
         self.assertSchema(data, JObj(sub_elems={
-            "login_url": JLeaf(str)
+            "login_url": JLeaf(str),
+            "is_create_cluster": JLeaf(bool)
```

Comment:
For consistency with the dashboard setting name, wouldn't be better to name it
``ìnit_create_cluster```?

Reply:
Done.